### ISSUE TITLE
Cannot comment on deleted lines or even track them in diff view in VSCode

### ISSUE THREAD

**Issue by TheQuantumPhysicist** [microsoft/vscode-pull-request-github#4038]
I only found an issue about deleted files #3421. Can't say this is unrelated, can't say this is the same.

- Extension version: v0.50.0
- VSCode Version: 1.72.0
- OS: Debian 11.5

Steps to Reproduce:

1. Create a pull request that removes code
2. Open the pull request in VSCode, and view the file with deleted lines
3. Attempt to add a comment on a deleted line in diff-view, you can't. The "plus" sign doesn't show to add a comment on deleted lines.
4. Even worse, if someone does it on github and adds a comment, it's not possible to view them properly on VSCode, and basically VSCode shows an empty file with the comment

**Comment by alexr00** [comment]
> Open the pull request in VSCode, and view the file with deleted lines

Where are you opening the file from? Do you have the pull request checked out? If you see this with a pull request in a public repo, can you share a link to the pull request?

**Comment by TheQuantumPhysicist** [comment]
I added you to that repo.

**Comment by alexr00** [comment]
>  I'm wondering whether you faced zero issues on your part

Correct, I faced no issues leaving and viewing comments on deleted lines. Thank you for the example repo, I'm looking into it.
